VoodooPS2Trackpad/VoodooPS2SynapticsTouchPad.cpp

```diff
@@ -190,7 +190,7 @@ ApplePS2SynapticsTouchPad* ApplePS2SynapticsTouchPad::probe(IOService * provider
     else
     {
         INFO_LOG("VoodooPS2Trackpad: Identify bytes = { 0x%x, 0x%x, 0x%x }\n", buf3[0], buf3[1], buf3[2]);
-        if (0x46 != buf3[1] && 0x47 != buf3[1])
```

Comment:
Is this the hack you mentioned you removed? CC @usr-sse2 what do you think about it?